server/src/routing.js

```diff
@@ -7,31 +7,39 @@ class Routing {
 
   routing () {
     const notConnected = this.app.session.notConnected.bind(this.app.session)
-    const connected = this.app.session.connected.bind(this.app.session)
-    const connectedOrBuilder = this.app.session.connectedOrBuilder.bind(this.app.session)
+    // const connected = this.app.session.connected.bind(this.app.session)
```

Comment:
Why is the following code commented out?

Reply:
It's right checking, so we can still access the routes until login pages are done